src/themes/mainTheme.ts

```diff
@@ -65,10 +66,38 @@ export default createMuiTheme({
     },
   },
   typography: {
-    fontFamily:
-      "-apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
+    fontFamily: 'Barlow,  sans-serif',
+    h1: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '3rem',
+    },
+    h2: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '2.5rem',
+    },
+    h3: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '2.1rem',
+    },
+    h4: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '1.8rem',
+    },
+    h5: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '1.5rem',
+    },
+    h6: {
+      fontFamily: 'Merriweather, serif',
+      fontSize: '1.2rem',
+    },
```

Comment:
jeg har egentlig mest lyst til å fase ut `Heading`, `SubHeading` og `NormalText`, og heller bare bruke `<Typography>` direkte med `variant` som prop (`h1`, `h2`, `body1`, osv)
Da kan Kine sette opp hvordan alle disse typene skal se ut (som vi skal bruke): https://material.io/develop/web/docs/typography, så trenger vi bare å forholde oss til det som står i theme. atm overstyrer vi font-size i `SubHeading` bl.a., så det er to kilder til hvordan teksten skal fremstå

Gir det mening @sarahse ?

Reply:
ja, jeg er enig i det 👍